roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -179,6 +179,11 @@ masterPublicURL: {{ openshift.master.public_api_url }}
 networkConfig:
   clusterNetworkCIDR: {{ openshift.master.sdn_cluster_network_cidr }}
   hostSubnetLength: {{ openshift.master.sdn_host_subnet_length }}
+{% if openshift.common.version_gte_3_7 | bool %}
+  clusterNetworks:
+  - cidr: {{ openshift.master.sdn_cluster_network_cidr }}
```

Comment:
This should be osm_cluster_network_cidr, as osm_cluster_network_cidr is now required.  We can remove the logic for openshift.master.sdn_cluster_network_cidr.